superset-frontend/src/components/URLShortLinkButton/index.jsx

```diff
@@ -50,7 +51,11 @@ class URLShortLinkButton extends React.Component {
 
   getCopyUrl(e) {
     e.stopPropagation();
-    getShortUrl(this.props.url)
+    getDashboardPermalink(
+      String(this.props.dashboardId),
+      {},
```

Comment:
This link is not considering the applied filters. The empty `filterState` was intentional?

<img width="479" alt="Screen Shot 2022-03-10 at 2 54 37 PM" src="https://user-images.githubusercontent.com/70410625/157725381-5f729c0e-e586-4eef-a8cb-ee107e2112e9.png">